docs/reference/modules/snapshots.asciidoc

```diff
@@ -329,6 +329,38 @@ POST /_snapshot/my_unverified_backup/_verify
 
 It returns a list of nodes where repository was successfully verified or an error message if verification process failed.
 
+[float]
+===== Repository Cleanup
+Repositories can over time accumulate data that is not referenced by any existing snapshot. This is a result of the data safety guarantees
```

Comment:
Mention that the cleanup functionality is also automatically run on snapshot deletion?

Reply:
Hmm I'm not sure. It would make sense now, but I was gonna add the logic to clean up all the individual shard folders in a follow-up which we can't run on delete (it's just super slow), not sure if it's worth explaining the difference here?

Reply:
Most folks shouldn't bother about this API, given that all relevant clean-up will happen on snapshot deletion. I do not want to raise the impression that you will *have* to run this API to sensibly operate  an ES cluster.

Reply:
Fair point, I added a note on that now in 
25fb827